features/upgrade/workloads/descheduler-upgrade.feature

```diff
@@ -48,7 +48,7 @@ Feature: Descheduler major upgrade should work fine
   @upgrade-check
   @users=upuser1,upuser2
   @destructive
-  @4.16 @4.15 @4.14 @4.13 @4.12 @4.11 @4.10 @4.9 @4.8 @4.7
+  @4.14 @4.13 @4.12 @4.11 @4.10 @4.9 @4.8 @4.7
```

Comment:
Perhaps we need to add the version back now ?

Reply:
yup, just testing it, will add once the test passes